lib/importers/schema_dot_org_json_data.rb

```diff
@@ -67,4 +68,9 @@ def process_organization_name(job)
     return nil
   end
 
+  def has_minimum_required_props(job_posting)
+    required_properties = ['hiringOrganization','datePosted', 'url']
+    required_properties.map {|x| !job_posting[x].blank?}.all?
```

Comment:
Phew.  Thought i was going nutso. 

They are similar but different, so i may have to keep them separate.  However, the required properties should be factored out so they reside in one place.